tooling/lib/test/karma.js

```diff
@@ -20,10 +19,6 @@ exports.test = async function test(options, packageName, files) {
 
   const cfg = makeConfig(packageName, options);
 
-  if (packageName === '@webex/plugin-phone' || packageName === '@webex/media-engine-webrtc') {
```

Comment:
If we changed this to the meetings plugin, would we get the injected profile on firefox?

Reply:
I fixed `openh264.js` (not .sh) and got it working, but it's still broken
It only injects mac so extra effort needs to be done to get it working for injecting into windows as well

Reply:
once we have anthony fix for H264 we should be fine

Reply:
The openH264 code is to inject the codec into a profile
This code has been broken for a while now so I'd rather just get rid of it instead of fixing it because it's needs to be refactored to also generate profiles for Windows